packages/eslint-plugin/lib/config/core.ts

```diff
@@ -1,158 +1,144 @@
+import js from "@eslint/js";
 import type { Linter } from "eslint";
-import { sourceFiles } from "../utils/patterns";
+import { concat } from "eslint-flat-config-utils";
+import globals from "globals";
+import { javaScriptFiles } from "../utils/patterns.ts";
 
-const config: Linter.Config = {
-    overrides: [
-        {
-            files: sourceFiles,
-            plugins: ["import"],
-            extends: [
-                "eslint:recommended"
-            ],
-            parserOptions: {
-                sourceType: "module",
-                ecmaVersion: "latest"
-            },
-            env: {
-                es2024: true,
-                node: true,
-                commonjs: true
-            },
-            rules: {
-                // eslint:recommended overwrite some rules
-                "no-cond-assign": ["error", "except-parens"],
-                "no-labels": ["warn", { allowLoop: true, allowSwitch: false }],
-                "no-prototype-builtins": "off",
+const config: Linter.FlatConfig[] = await concat(
+    js.configs.recommended as Linter.FlatConfig,
+    {
+        name: "Workleap/Core",
```

Comment:
Is "Workleap/Core" matching a recommended format pushed by eslint?

If not, could you update to lowercase and use "-" between words please? 🙏🏻 Ex.: "workleap-core"

Reply:
Also, I wonder why the "by-project-type" don't have a "name" ?

Reply:
The slash is the recommended name, and this is only used in debug view, and also in the inspector. The by project type ones don't have their own custom rules, so there is nothing to name. I had at one point toyed around with a function that did a deep rewrite of the names so that it appended the by project name to everything, but it seemed like a lot of work for not a lot of benefit.

Reply:
Got it thanks!

Could you name like: "workleap/core" thought? Feels more JS and less .NET hehe